src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractLocalFunctionTests.cs

```diff
@@ -1680,9 +1680,9 @@ class C
     public static void Main(string[] args)
     {
         bool Local() => args == null;
-        {|Rename:NewMethod|}(args);
```

Comment:
>args [](http://example.com/codeflow?start=29&length=4)

Why this change? Has it stopped working?

Reply:
> Why this change? Has it stopped working?

I think this is a change to the better. The refactoring no longer adds an unused parameter. This is due to an improvement in region analysis around parameters of enclosing methods.